prep-printsrv.sh

```diff
@@ -15,7 +15,13 @@ bash -x $tempscript
 cat teams.json | jq -r '.[] | ("cp teams/" + .id +  "/photo.jpg printsrv/Pictures/team" + .id + ".jpg")' > $tempscript
 bash -x $tempscript
 
+cat teams.json | jq -r '.[] | ("cp teams/" + .id +  "/photo.jpg printsrv/Pictures/team" + .id + ".jpg")' > $tempscript
+bash -ex $tempscript
+
```

Comment:
Ah, need to remove these lines, they got duplicated on the merge conflict fixing